ui/components/component-library/tag-url/README.mdx

```diff
@@ -16,55 +16,70 @@ The `TagUrl` accepts all props below as well as all [Box](/docs/ui-components-ui
 
 <ArgsTable of={TagUrl} />
 
-### Src
+### Action Button Label
 
-Use the `src` prop with an image url to render the `AvatarFavicon`.
+If we want to have a button in `TagUrl` component. You can update the `ButtonLink` component props used for the action button using `actionButtonProps`
 
 <Canvas>
-  <Story id="ui-components-component-library-tag-url-tag-url-stories-js--src" />
+  <Story id="ui-components-component-library-tag-url-tag-url-stories-js--action-button-label" />
 </Canvas>
 
 ```jsx
 
 import { TagUrl } from '../../ui/component-library/tag-url';
 
-  <TagUrl src="https://peepeth.com/favicon-32x32.png" />
-  <TagUrl src="https://1inch.exchange/assets/favicon/favicon-32x32.png"/>
-  <TagUrl src="https://uniswap.org/favicon.ico" />
+<TagUrl label="https://app.uniswap.org" actionButtonLabel="Permissions"/>
 
 ```
 
-### Label
+### Show Lock Icon
 
-Use the `label` prop for the text content of the `TagUrl` component
+Use the `showLockIcon` prop to render a lock icon next to the `label`. It's intended use is to display if the url uses `https`. Props for the lock icon can be changed using the `lockIconProps`.
 
 <Canvas>
-  <Story id="ui-components-component-library-tag-url-tag-url-stories-js--label" />
+  <Story id="ui-components-component-library-tag-url-tag-url-stories-js--show-lock-icon" />
 </Canvas>
 
 ```jsx
 
 import { TagUrl } from '../../ui/component-library/tag-url';
 
-  <TagUrl label="https://widget.getacute.io" />
-  <TagUrl label="app.uniswap.org" />
-  <TagUrl label="https://metamask.github.io" />
+<TagUrl label="https://app.uniswap.org" showLockIcon/>
 
 ```
 
-### Action Button Label
+### Src
 
-If we want to have a button in `TagUrl` component. You can update the `ButtonLink` component props used for the action button using `actionButtonProps`
+Use the `src` prop with an image url to render the `AvatarFavicon`.
 
 <Canvas>
-  <Story id="ui-components-component-library-tag-url-tag-url-stories-js--action-button-label" />
+  <Story id="ui-components-component-library-tag-url-tag-url-stories-js--src" />
 </Canvas>
 
-### Show Lock Icon
+```jsx
 
-Use the `showLockIcon` prop to render a lock icon next to the `label`. It's intended use is to display if the url uses `https`. Props for the lock icon can be changed using the `lockIconProps`.
+import { TagUrl } from '../../ui/component-library/tag-url';
+
+  <TagUrl src="https://peepeth.com/favicon-32x32.png" />
+  <TagUrl src="https://1inch.exchange/assets/favicon/favicon-32x32.png"/>
+  <TagUrl src="https://uniswap.org/favicon.ico" />
+
+```
```

Comment:
Formatting and import statement. Not added a fall back component. Will note in stories file too

```jsx
import { TagUrl } from '../../ui/component-library';

<TagUrl
  label="peepeth.com"
  src="https://peepeth.com/favicon-32x32.png"
/>
<TagUrl
  label="1inch.exchange"
  src="https://1inch.exchange/assets/favicon/favicon-32x32.png"
/>
<TagUrl
  label="app.uniswap.org"
  src="https://uniswap.org/favicon.ico"
/>
<TagUrl label="fallback" src="" />
```